systemtest/src/test/java/io/strimzi/systemtest/specific/ClusterOperationST.java

```diff
@@ -0,0 +1,114 @@
+        List<String> topicNames = IntStream.range(0, 5).boxed().map(i -> "test-topic-" + i).collect(Collectors.toList());
+        List<String> producerNames = IntStream.range(0, 5).boxed().map(i -> "hello-world-producer-" + i).collect(Collectors.toList());
+        List<String> consumerNames = IntStream.range(0, 5).boxed().map(i -> "hello-world-consumer-" + i).collect(Collectors.toList());
+        List<String> continuousConsumerGroups = IntStream.range(0, 5).boxed().map(i -> "continuous-consumer-group-" + i).collect(Collectors.toList());
+        int continuousClientsMessageCount = 300;
+        List<Node> nodes = kubeClient().getClusterWorkers();
+
+        KafkaResource.kafkaPersistent(CLUSTER_NAME, 3, 3)
+                .editOrNewSpec()
+                .editEntityOperator()
+                .editUserOperator()
+                .withReconciliationIntervalSeconds(30)
+                .endUserOperator()
+                .endEntityOperator()
+                .endSpec()
+                .done();
+
+        topicNames.forEach(topicName -> KafkaTopicResource.topic(CLUSTER_NAME, topicName, 3, 3, 2).done());
+
+        String producerAdditionConfiguration = "delivery.timeout.ms=20000\nrequest.timeout.ms=20000";
+        producerNames.forEach(producerName -> KafkaClientsResource.producerStrimzi(
+                producerName,
+                KafkaResources.plainBootstrapAddress(CLUSTER_NAME),
+                topicNames.get(producerNames.indexOf(producerName)),
+                continuousClientsMessageCount, producerAdditionConfiguration).done());
+
+
+        consumerNames.forEach(consumerName -> KafkaClientsResource.consumerStrimzi(
+                consumerName,
+                KafkaResources.plainBootstrapAddress(CLUSTER_NAME),
+                topicNames.get(consumerNames.indexOf(consumerName)),
+                continuousClientsMessageCount, "",
+                continuousConsumerGroups.get(consumerNames.indexOf(consumerName))).done());
+
+        // ##############################
+        // Nodes draining
+        // ##############################
+        for (Node node : nodes) {
+            drainNode(node.getMetadata().getName());
+            setNodeSchedule(node.getMetadata().getName(), true);
+        }
+
+        producerNames.forEach(producerName -> ClientUtils.waitTillContinuousClientsFinish(producerName, consumerNames.get(producerName.indexOf(producerName)), NAMESPACE, continuousClientsMessageCount));
+        producerNames.forEach(producerName -> kubeClient().getClient().batch().jobs().inNamespace(NAMESPACE).withName(producerName).delete());
+        consumerNames.forEach(consumerName -> kubeClient().getClient().batch().jobs().inNamespace(NAMESPACE).withName(consumerName).delete());
+    }
+
+    @BeforeAll
+    void setup() {
+        ResourceManager.setClassResources();
+        prepareEnvForOperator(NAMESPACE);
+
+        applyRoleBindings(NAMESPACE);
+        BundleResource.clusterOperator(NAMESPACE).done();
```

Comment:
Does it means we cannot use OLM/Helm install for this scenario?

Reply:
just copied :) I will fix it